concepticondata/conceptlists.tsv

```diff
@@ -1,4 +1,5 @@
 ID	AUTHOR	YEAR	LIST_SUFFIX	ITEMS	TAGS	SOURCE_LANGUAGE	TARGET_LANGUAGE	URL	REFS	PDF	NOTE	PAGES	ALIAS
+Sun-1991-1004	Sun, Hongkai et al.	1991		1004	basic	English	Tibeto-Burman languages	http://stedt.berkeley.edu/~stedt-cgi/rootcanal.pl	Sun1991		This is a list of concepts generated from the entries listed in the STEDT database; there are many concepts overlapping, some of which are only used for a single doculect.	
```

Comment:
I propose to rewrite the note as:

"This concept list originally served as a questionnaire for a large-scale investigation on Tibeto-Burman languages. The original questionnaire was in Chinese with no English translation. Later the [STEDT](http://stedt.berkeley.edu) project ([Matisoff 2015](:bib:Matisoff2015)) digitized the data, translating Chinese concept labels to English, but not listing the original Chinese forms. As a result, some concept labels are identical, although they are different in the Chinese version. We are still trying to add the original Chinese concept labels to this resource, but for the moment, we only link the STEDT version, occasionally adding Chinese concept labels, where we figure they are important to distinguish the original meaning."

I also suggest to not label this as "basic" but rather as "questionnonnaire" as we did with Buck and other lists, it is surely not a "basic" list.

The reference for Matisoff, I suggest to add:
```
@Book{Matisoff2015,
  Title                               = {{T}he {S}ino-{T}ibetan {E}tymological {D}ictionary and {T}hesaurus project},
  Address                             = {Berkeley},
  Author                              = {Matisoff, James A.},
  Publisher                           = {University of California},
  Year                                = {2015},

  Owner                               = {mattis},
  Timestamp                           = {2016.09.08}
}

```